tools/orc/main.go

```diff
@@ -531,6 +556,84 @@ func showTdxComponent(indent string, bnd *bundle.Bundle, comp *bundle.Component)
 	fmt.Printf("%s  Memory:  %d MiB\n", indent, comp.TDX.Resources.Memory)
 }
 
+// sgxGenerateKey generates a 3072-bit RSA key with public exponent 3 as required for SGX.
```

Comment:
I thought that small exponents should be avoided. But if they required 3, then what can you do? :shrug:

Reply:
It doesn't really matter as MRSIGNER is not used anywhere in our system (we just compare against a reference value in the policy). So it doesn't really matter even if this key is compromised. Which is why we are adding these ephemeral keys here.